src/services/symbolication.rs

```diff
@@ -1642,8 +1642,12 @@ impl SymbolicationActor {
                     procspawn::serde::Json(cfi_caches),
                     minidump.clone(),
                     spawn_time,
+                    options.compare_stackwalking_methods,
                 ),
-                |(cfi_caches, minidump, spawn_time)| -> Result<_, ProcessMinidumpError> {
+                |(
+                    cfi_caches, minidump,
+                    spawn_time,
+                    compare_stackwalking_methods)| -> Result<_, ProcessMinidumpError> {
```

Comment:
dear rustfmt, what is this?  🤣

Reply:
I know, you hate to see it. For some reason `cargo fmt` refuses to touch the file.

Reply:
maybe extract that as a clearly typed and named method instead of a closure would help with this as well.

Reply:
I can extract it, but I think we're starting to run out of sensible method names. We already have `stackwalk_minidump_with_cfi` and `do_minidump_stackwalk`; this one would probably need a similar name.